packages/shared_preferences/lib/shared_preferences.dart

```diff
@@ -70,6 +70,9 @@ class SharedPreferences {
   /// String.
   String getString(String key) => _preferenceCache[key];
 
+  /// Returns true if persistent storage the contains the given [key].
+  bool contains(String key) => _preferenceCache.containsKey(key);
```

Comment:
I think this should be named `containsKey` for consistency with the `Map` API. (In retrospect we probably should have just make `SharedPreferences` implement `Map` but that's out of scope for this change.)

Reply:
yuo right! done!